core/services/job/runner_integration_test.go

```diff
@@ -345,8 +343,8 @@ ds1 -> ds1_parse;
 		os.MaxTaskDuration = models.Interval(cltest.MustParseDuration(t, "1s"))
 		err = jobORM.CreateJob(context.Background(), &os, os.Pipeline)
 		require.NoError(t, err)
-		var jb job.SpecDB
-		err = db.Preload("OffchainreportingOracleSpec", "id = ?", os.ID).
+		var jb = job.SpecDB{IDEmbed: job.IDEmbed{os.ID}}
+		err = db.Preload("OffchainreportingOracleSpec").
 			Find(&jb).Error
```

Comment:
Should we have a `First` here not a Find?

This query is weird.

Reply:
Why not just a `First` with `Where`?

Reply:
The Find will use the PK, but First/Where is more clear